springwolf-core/src/test/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/operationdata/annotation/AsyncPublisherAnnotationScannerTest.java

```diff
@@ -66,7 +66,7 @@ public void scan_componentHasPublisherMethod() {
         Message message = Message.builder()
                 .name(SimpleFoo.class.getName())
                 .title(SimpleFoo.class.getSimpleName())
-                .description(null)
+                .description("")
```

Comment:
One of the issues that didn't come up because of the equals, the default value for the description is empty string